notebooks/05_operationalize/pipeline-demo/als_deep_dive.py

```diff
@@ -0,0 +1,380 @@
+
```

Comment:
this is a converted version of the als_deep_dive notebook? what is it used for?

Reply:
This is the script with little modification that can run on ADB. AML pipeline will create ADB cluster to run this script , which can be either on ADB workspace or local machine.